cmake/Buildwhole-body-controllers.cmake

```diff
@@ -8,9 +8,9 @@ include(FindOrBuildPackage)
 find_or_build_package(WBToolbox QUIET)
 find_or_build_package(qpOASES QUIET)
 
-ycm_ep_helper(WBIToolboxControllers TYPE GIT
+ycm_ep_helper(whole-body-controllers TYPE GIT
               STYLE GITHUB
-              REPOSITORY robotology-playground/WBI-Toolbox-controllers.git
+              REPOSITORY robotology-playground/whole-body-controllers.git
```

Comment:
This can be moved to `robotology` now: https://github.com/robotology/whole-body-controllers/issues/22 .